src/utilities/oas-schema/oas-schema.ts

```diff
@@ -72,30 +72,30 @@ class OASSchema {
               (parameter) =>
                 parameter.examples && parameter.examples[groupName],
             )
-            .map((parameter) => {
+            .reduce((groups, parameter) => {
               const { name, examples } = parameter;
-              return [name, examples[groupName]];
-            });
+              return {
+                ...groups,
+                [name]: examples[groupName].value,
+              };
+            }, {});
 
           return ParameterWrapper.wrapParameters(
-            Object.assign(
-              {},
-              Object.fromEntries(requiredParametersAndExamples),
-              Object.fromEntries(groupParameters),
-            ),
+            Object.assign({}, requiredParametersAndExamples, groupParameters),
             groupName,
           );
         });
       } else {
         parameters = ParameterWrapper.wrapParameters(
-          Object.fromEntries(requiredParametersAndExamples),
+          requiredParametersAndExamples,
         );
       }
 
-      return [method.operationId, parameters];
-    });
-
-    return Object.fromEntries(operationIdToParameters);
+      return {
+        ...parameterExamples,
+        [method.operationId]: parameters,
+      };
+    }, {});
```

Comment:
`getParameters` has become pretty long, at nearly 50 lines now. Not necessarily part of this PR, but what are your thoughts on a ticket to reduce some of the complexity here by breaking up this function?

Reply:
that's a great callout! Christine and I have actually been talking through a general refactoring of some stuff that will hopefully help reduce complexity